helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -173,10 +192,19 @@ private String determineHeliosHost(final Info dockerInfo) throws HeliosDeploymen
 
     if (dockerHostAddressIsLocalhost()) {
       if (isDockerForMac(dockerInfo)) {
+        log.info("determineHeliosHost: local environment appears to be Docker for Mac");
         try {
-          log.info("determineHeliosHost: local environment appears to be Docker for Mac");
-          return InetAddress.getLocalHost().getHostAddress();
-        } catch (UnknownHostException e) {
+          InetAddress localhost = InetAddress.getLocalHost();
+          if (localhost.isLoopbackAddress()) {
```

Comment:
@davidxia @mattnworb updated logic so that we only guess the IP when we find loopback